$OpenBSD: patch-src_config_file_c,v 1.1 2020/10/13 15:58:06 cwen Exp $

Allow to properly read the configuration files on archs where char is unsigned
(arm, ppc) as EOF is usually -1.
See https://github.com/nickg/xcowsay/commit/666ad99

Index: src/config_file.c
--- src/config_file.c.orig
+++ src/config_file.c
@@ -78,7 +78,7 @@ static token_t next_token(FILE *f, strbuf_t* sbuf, int
    bool skip_to_eol = false;
    bool in_quotes = false;
    for (;;) {
-      char next = fgetc(f);
+      int next = fgetc(f);
       if (EOF == next)
          return (has_chars(sbuf) && !skip_to_eol) ? tTOKEN : tEOF;
       else if ('\n' == next) {
